packages/devtools_app/lib/src/service/vm_flags.dart

```diff
@@ -34,7 +34,7 @@ class VmFlagManager extends Disposer {
   }
 
   Future<void> _initFlags() async {
-    final flagList = await service!.getFlagList() ?? FlagList(flags: []);
+    final flagList = await service?.getFlagList() ?? FlagList(flags: []);
```

Comment:
can _service be late and non-nullable instead? that way service can be non-nullable

Reply:
done